### DISCUSSION TITLE
meeting name and info font too light

### DISCUSSION THREAD

**Discussion by dist10** [code4recovery/12-step-meeting-list#1156] [Q&A] [answered]
Just installed 12 step meeting list plugin and entered two meetings. In both the map shows fine as does the location information. However, when previewing, the Meeting name, get directions, and meeting information fonts are too light to read. Using Elementor. Unable to correct in theme settings. Any ideas? https://dist10org.fatcow.com/dev/meetings/allyn-historical-church-sunday-700-pm/

**Comment by robyng** [comment, marked as answer]
Hi @dist10 thanks for posting. 12 Step Meeting List (TSML) plugin inherits font colors from what is set in your theme. I see you have a light color set for headings and paragraphs and that looks great on dark background like on your https://dist10org.fatcow.com/dev/trusted-servants/ page. On your blog page it looks too light like on the meetings page. Try adding custom css for the meetings pages. A couple approches I see is change font color on TSML or adding background color to make white text readable. To change font color you can try this in your CSS: 

`#tsml {
    color: black;
}`

**Comment by dist10** [comment]
Thank you Robin, I added the CSS code in and it works beautifully now.
I only just took over as webservant. The initial developer of the site used
Elementor and I am still not fully comfortable with it. Wordpress Classic
and sometimes block make more sense to me.

On to more data entry. Thanks again.

Ron
Dist10 webservant

**Reply by robyng** [reply]
Great glad to hear Ron! That's a great way to be of service. I'm sure you will find a way that is manageable for you. I've ran into similar situation where it was faster for me to recreate the pages in a builder I was comfortable with than to learn a new one. Good luck!

**Comment by dist10** [comment]
I usually do that too Robyn, but in this case we're talking about a lot of
pages. I really don't relish taking over someone else's work.

The meetings are showing up beautifully now. It has been quite a time
consuming task with lots of meetings.

The only glitches I found were in entering a phone contact number in the
contact area. Kept asking me to open an app to call. I moved them to
remarks section. The previous webmaster created individual pages for each
group and I have been posting a link to them in that contact area under web
addresses. Seems to work but need to get the word out.

Thanks again for all your help.
Ron
